Fix vale warnings in End-user Guide (#2078)
codenvy/che-docs: modules/end-user-guide/partials/ref_devfile-reference.adoc

* `CHE_MACHINE_AUTH_SIGNATURE__ALGORITHM`: The encryption algorithm used in the secured communication with the {prod-short} server.

> A devfiles may only need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs (for example as part of a devfile command). The environment variables used to securely access the {prod-short} server are out of scope for devfiles and are available only for advanced use cases, which are handled by the {prod-short} plug-ins.

Reviewer: `A devfiles`?
may > might
IBM SG:
```
may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead.
```

Reviewer: ```suggestion
A devfile might need the `CHE_PROJECTS_ROOT` environment variable to locate the cloned projects in the component's container. More advanced devfiles might use the `CHE_WORKSPACE_LOGS_ROOT__DIR` environment variable to read the logs. The environment variables for securely accessing the {prod-short} server are out of scope for devfiles. These variables are available only to {prod-short} plug-ins, which use them for advanced use cases.
```

Reviewer: IBM SG: Use parentheses to identify items such as abbreviations, symbols, and measurements, but avoid using parentheses in general text.

Reviewer: Is `(for example as part of a devfile command)` redundant information?

Reviewer: IBM SG: Use simple verbs and tenses. Use simple language. Avoid needless words. Try to keep
sentences to an average of 32 words or fewer, and vary sentence lengths.

      app: postgres
----

> Additionally, it is also possible to modify the entrypoints (command and arguments) of the containers in the resource list. 

Reviewer: entry points?
```suggestion
Additionally, you can modify the entrypoint commands and arguments of the containers in the resource list. 
```

Author: @MichalMaler entrypoints is used in the procedure as one word, so I assume that is correct. However, sometimes it is capitalized and sometimes it isn't, is that intended? I don't know enough about it to decide, if we can safely remove the brackets without altering the meaning. Can you please take a look?

Reviewer 2: will do!

Reviewer 2: Regarding what I got from developers, there is no need to capitalize this. Then, the correct expression coming from the Kubernetes slang is `entrypoint`. On top of that, we are talking about something from the devfile reference, so it is not to be mentioned as `entry point`.
